packages/system/trust-manager/Makefile

```
@@ -0,0 +1,14 @@
+export NAME=trust-manager
+export NAMESPACE=cozy-cert-manager
+
+include ../../../hack/package.mk
+
+update:
```

checkmake (0.3.2)

[warning] Target body for "update" exceeds allowed length of 5 lines (8).

(maxbodylength)